server/index.js

```diff
@@ -8,28 +8,131 @@ const path = require('path');
 const { QueryTypes } = require('sequelize');
 const sequelize = require('./database/index.js');
 const cors = require('cors');
+const bodyParser = require('body-parser');
 
 app.use(cors());
-
+app.use(bodyParser());
 app.use(express.static(path.join(__dirname, '../client/public/dist')))
 
 
+// ------------------------------------------------------- //
+// ------------- SDC ADDITIONS TO CODEBASE --------------- //
+// ------------------------------------------------------- //
+
+// to display content based on product ID in URL path e.g. localhost:3002/4
+app.use('/:productId', express.static(path.join(__dirname, '../client/public/dist')))
+
+// helper functions
+const findProduct = async function(prodId) {
+  const productData = await db.Product.findAll({
+    where: {
+      id: prodId
+    },
+    attributes: {exclude: ['createdAt', 'updatedAt']}
+  })
+  return productData
+}
+
+const updateProduct = function(prodId, prodInfo) {
+  db.Product.update({
+    name: prodInfo.name,
+    price: prodInfo.price,
+    reviews: prodInfo.reviews,
+    reviewCount: prodInfo.reviewCount
+  }, {
+    where: {
+      id: prodId
+    }
+  });
+}
+
```

Comment:
Would we need any kind of error handling or notification in the update and delete functions?

Reply:
yes! that's something I had planned to lookup and implement but slipped my mind. not sure why it's not something I know but yeah, great point. i'll re-submit afterwards